azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -1352,6 +1358,17 @@ private void addInitContainers(final int executionId,
             dependency + " in versionSet");
       }
     }
+    if (this.prefetchAllCredentials) {
+      try {
+        final String imageFullPath =
+            versionSet.getVersion("security-init").get().pathWithVersion();
```

Comment:
Consider replacing "security-init" here to InitContainerType.SECURITY.initPrefix to avoid duplicate const strings